libraries/AP_Compass/AP_Compass_Params.cpp

```diff
@@ -130,6 +130,13 @@ const AP_Param::GroupInfo AP_Compass_Params::var_info[] = {
     // @Increment: 1
     // @User: Advanced
     AP_GROUPINFO("MOT",    9, AP_Compass_Params, motor_compensation, 0),
+
+    // @Param: BAT_IDX
+    // @DisplayName: Motor interference compensation battry moniter index
+    // @Description: Motor interference compensation source battry moniter index
+    // @Values: 0:First battery, 1:Second battery
```

Comment:
Short a lot of values here. Arguably shouldn't we be looking at the same parameter used in the AP_Motors library? It's odd that this results in having to set the same parameter in both locations...

Reply:
Planes might not want to use the VTOL battery. The values are just a copy of the motors one... but your right it is 7 monitors short.

Reply:
That's a fair point. Usually the VTOL is the problem, but that's a fair point here. (Really VTOL actually want's to compensate for both, but we don't support that yet).